iidm/iidm-xml-converter/src/test/resources/V1_1/LccRoundTripRef.xml

```diff
@@ -20,8 +20,9 @@
                 <iidm:switch id="BK2" name="Breaker" kind="BREAKER" retained="true" open="false" node1="3" node2="4"/>
                 <iidm:switch id="DISC_BBS1_BK3" name="Disconnector" kind="DISCONNECTOR" retained="false" open="false" node1="0" node2="5"/>
                 <iidm:switch id="BK3" name="Breaker" kind="BREAKER" retained="true" open="false" node1="5" node2="6"/>
+                <iidm:bus nodes="0,1,2,3,4,5,6"/>
```

Comment:
If there is no V and/or theta, should we serialize buses?

Reply:
Line deleted.